htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -971,13 +969,13 @@ function print_left_eldy_menu($db, $menu_array_before, $menu_array_after, &$tabM
 				$shorturl = $shorturl.($param ? '?'.$param : '');
 			}
 
-
-			print '<!-- Process menu entry with mainmenu='.$menu_array[$i]['mainmenu'].', leftmenu='.$menu_array[$i]['leftmenu'].', level='.$menu_array[$i]['level'].' enabled='.$menu_array[$i]['enabled'].', position='.$menu_array[$i]['position'].' -->'."\n";
+			// For debug only ? (add global option) ?
+			//print '<!-- Process menu entry with mainmenu='.$menu_array[$i]['mainmenu'].', leftmenu='.$menu_array[$i]['leftmenu'].', level='.$menu_array[$i]['level'].' enabled='.$menu_array[$i]['enabled'].', position='.$menu_array[$i]['position'].' -->'."\n";
 
 			// Menu level 0
 			if ($menu_array[$i]['level'] == 0) {
 				if ($menu_array[$i]['enabled']) {     // Enabled so visible
-					print '<div class="menu_titre">'.$tabstring;
+					print '<li class="menu_titre">'.$tabstring;
```

Comment:
Using li instead of div for menu is a good idea.
However, we must follow the rule 1 PR = 1 feature purpose. Can you make a new PR with only this goal in mind into a separate atomic PR.